src/main/java/appeng/items/tools/powered/ToolPortableCell.java

```diff
@@ -186,4 +186,9 @@ public IGuiItemObject getGuiObject( final ItemStack is, final World w, final Blo
 	{
 		return new PortableCellViewer( is, pos.getX() );
 	}
+
+	@Override
+	public boolean shouldCauseReequipAnimation(ItemStack oldStack, ItemStack newStack, boolean slotChanged) {
```

Comment:
Will make the changes, thanks for the review. Tossed this together quick after noticing the issue and figuring fixing it myself would be faster than opening a new issue, just forgot code style. I'll get on that.